packages/date-picker/src/FilterDateRangePicker/utils/validateEndDateBeforeStartDate.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { validateEndDateBeforeStartDate } from "./validateEndDateBeforeStartDate"
+
+describe("validateEndDateBeforeStartDate", () => {
+  test("end date after start date", () => {
+    const startDate = new Date("2022-04-03")
+    const startDateFieldLabel = "Start date"
+    const endDate = new Date("2022-05-01")
+    const endDateInputValue = "01/05/2022"
+    const { validationResponse, newDate } = validateEndDateBeforeStartDate({
+      startDate,
+      startDateFieldLabel,
+      endDate,
+      endDateInputValue,
+    })
+
+    expect(validationResponse).toStrictEqual({
+      date: endDate,
+      inputValue: endDateInputValue,
+      status: undefined,
+      validationMessage: undefined,
+      isInvalid: false,
+      isDisabled: false,
+      isEmpty: false,
+      isValidDate: true,
+    })
+
+    expect(newDate).toBe(endDate)
+  })
+
+  test("end date equals start date", () => {
```

Comment:
Confirming we didn't want to have an opinion on this and leave it up to the consumer if they wanted to make this an error or warning

Reply:
Yeah I think so. It's a valid selection for react-day-picker, so I think it makes sense to replicate it. Better than over-restricting :)

Reply:
Yep makes sense!